README.md

```diff
@@ -57,6 +57,15 @@ If you have an existing workflow that runs your tests you can just add the `Uplo
 | coverageEndpoint | The remote endpoint to upload coverage data to                         |                      | No       |
 | coverageToken    | A token to authenticate with the remote endpoint and identify the repo |                      | No       |
 
+This action also can accept a markdown comment . Modify your workflow like so:
+
+```json
+  customMessage:
+     description: toggle if the coverageEndpoint expects to respond with the custom message or just the difference
+     default: 'comment'
+     required: false
+```
```

Comment:
I think using a `yml` markdown instead of a `json` markdown is better here since it's supposed to be a modification for the workflow.

Using `yml` here should also remove the red highlighting that shows up. I think `json` markdowns need the open and close curly brackets or else it will highlight red.